ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-caseworkerEvents-nonprod.json

```diff
@@ -12,8 +12,14 @@
       },
       {
         "UserRoles": [
-          "caseworker-civil-solicitor",
-          "judge-profile"
+          "CITIZEN-CLAIMANT-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE",
+          "cui-nbc-profile",
+          "hearing-schedule-access",
+          "judge-profile",
+          "legal-adviser",
```

Comment:
is GS_profile not enough ? do we really need  "judge-profile", "legal-adviser",  etc

Reply:
I've changed over to GS_profile